mkdocs.yml

```diff
@@ -15,11 +15,13 @@ nav:
     - Server Behavior: 'server-behavior.md'
 
 markdown_extensions:
+  - admonition
```

Comment:
This enables `!!! note` on the docs. Reference: https://squidfunk.github.io/mkdocs-material/reference/admonitions/